openfl-docker/Dockerfile.base

```diff
@@ -13,28 +14,26 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN --mount=type=cache,id=apt-dev,target=/var/cache/apt \
     apt-get update && \
     apt-get install -y \
+    git \
     python3-pip \
     python3.10-dev \
     ca-certificates \
     build-essential \
-    git \
     --no-install-recommends && \
+    apt-get purge -y linux-libc-dev && \
     rm -rf /var/lib/apt/lists/*
-RUN apt-get purge -y linux-libc-dev
 
 # Create an unprivileged user.
 RUN groupadd -g 1001 default && \
-    useradd -m -u 1001 -g default openfl
-USER openfl
+    useradd -m -u 1001 -g default user
+USER user
+WORKDIR /home/user
+ENV PATH=/home/user/.local/bin:$PATH
 
 # Install OpenFL.
-WORKDIR /home/openfl
-COPY --chown=openfl:default . .
-ENV PATH=/home/openfl/.local/bin:$PATH
+ARG OPENFL_REVISION=https://github.com/securefederatedai/openfl.git@ad53a0fdbefde8bc18404b6ea4d48344956b96ee
```

Comment:
Can we be more explicit, and reference the tag instead, as in:
```bash
https://github.com/securefederatedai/openfl.git@v1.6
```

Reply:
Fixed https://github.com/securefederatedai/openfl/pull/1096/commits/21edded270d3bc9769fb27f22198d144b3ff1b6d